packages/contracts/deploy/new/30_plugins/00_plugin-setups/13_token_voting_setup_zksync_conclude.ts

```diff
@@ -0,0 +1,40 @@
+import {TokenVotingSetup__factory} from '../../../../typechain';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {setTimeout} from 'timers/promises';
+import {skipIfNotZkSync} from '../../../helpers';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log(`Concluding token voting setup deployment.\n`);
+  const [deployer] = await hre.ethers.getSigners();
+
+  const {deployments, network} = hre;
+
+  const TokenVotingSetupDeployment = await deployments.get('TokenVotingSetup');
+  const tokenVotingSetup = TokenVotingSetup__factory.connect(
+    TokenVotingSetupDeployment.address,
+    deployer
+  );
+
+  // add a timeout for polygon because the call to `implementation()` can fail for newly deployed contracts in the first few seconds
+  if (network.name === 'polygon') {
+    console.log(`Waiting 30secs for ${network.name} to finish up...`);
+    await setTimeout(30000);
+  }
```

Comment:
Since this should be skipped if the network is not ['zksync'](https://github.com/aragon/osx/blob/zksync-testing/packages/contracts/utils/zkSync.ts), couldn’t this be removed ?

Reply:
Yes. Removed in https://github.com/aragon/osx/pull/596/commits/f0756ac0602f3e8d312c1c211137fc235d96ba6b